app/models/auction.rb

```diff
@@ -10,8 +12,13 @@ class Auction < ActiveRecord::Base
   # Disable STI
   self.inheritance_column = :__disabled
 
+  validates :delivery_deadline, presence: true
```

Comment:
the issue requested it be validated on create: https://github.com/18F/micropurchase/issues/549

do you think we should only validate on publish?